reviews/2020-opentelemetry-annual.md

```diff
@@ -0,0 +1,108 @@
+## Background
+OpenTelemetry is an integrated set of APIs and libraries that generate, collect, and describe telemetry about distributed systems. This data includes basic context propagation, distributed traces, metrics, and other signals in the future. OpenTelemetry makes it easy to get critical telemetry data out of your services: for each language it offers a single set of APIs, libraries, and data specifications, and developers can make use of which components they see fit.
+
+OpenTelemetry is well-suited to diverse software systems, from client applications, large monoliths, or highly-distributed microservices. It also provides a collector component that can proxy, aggregate, and enrich telemetry before it is exported to a backend.
+
+Supported languages :
+- .NET (SDK  and Auto-Instrumentation)
+- C/C++ SDK
+- GoLang SDK
+- Java (SDK and Auto-Instrumentation)
+- JavaScript SDK
+- Python SDK
+- Ruby SDK
+- Erland/Elixir SDK
+- PHP SDK
+- Rust SDK
+- Swift SDK
```

Comment:
Could you insert a link to the original Sandbox presentation? Similar to how it was done here https://github.com/cncf/toc/pull/440/files#diff-e9dc5bcb93599ba622b4e3e626fdd4eeR24

Reply:
@ccaraman links:

- Slides: https://docs.google.com/presentation/d/1roUFtkiVw25N7nRuUnMa-ziNhb6vzhZAe43jpBfiDiw/edit#slide=id.g5724df7a45_0_258
- Proposal: https://github.com/cncf/toc/blob/f4ecf1bf391867d0840434b638cd9ef6cb2f57fe/proposals/sandbox/opentelemetry.adoc
- PR: https://github.com/cncf/toc/pull/233